component/vdr/indy/go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/hyperledger/aries-framework-go-ext/component/vdr/indy
+
+go 1.15
+
+require (
+	github.com/btcsuite/btcutil v1.0.1
+	github.com/flimzy/diff v0.1.7 // indirect
+	github.com/flimzy/testy v0.1.17 // indirect
+	github.com/go-kivik/couchdb v2.0.0+incompatible // indirect
+	github.com/go-kivik/kivik v2.0.0+incompatible // indirect
+	github.com/go-kivik/kiviktest v2.0.0+incompatible // indirect
+	github.com/go-sql-driver/mysql v1.5.0 // indirect
+	github.com/gopherjs/gopherjs v0.0.0-20200217142428-fce0ec30dd00 // indirect
+	github.com/hyperledger/aries-framework-go v0.1.5-0.20201119203645-6dedee7a40f7
+	github.com/hyperledger/indy-vdr/wrappers/golang v0.0.0-20201031155907-5f437d26ed71
+	github.com/mr-tron/base58 v1.2.0
```

Comment:
the framework currently brings in another base58 library: https://github.com/hyperledger/aries-framework-go/blob/master/go.mod#L12